app/controllers/meetings/publish/notulen.js

```diff
@@ -68,7 +69,14 @@ export default class MeetingsPublishNotulenController extends Controller {
     const id = this.model.id;
     const response = yield fetch(`/prepublish/behandelingen/${id}`);
     const json = yield response.json();
-    return json.map((response) => response.data.attributes);
+
+    return json.map((response) => response.data.attributes)
+            .map((behandeling) => {
+              return {
+                ...behandeling,
+                content: htmlSafe(behandeling.content),
```

Comment:
Maybe it would be better to install / write an html-safe helper so this can be done in the template where needed.

Reply:
quite sure we have a html-safe helper and use it elsewhere, so that's definitely an option :+1:

Reply:
Woops, not sure how I missed that :smile:.

Reply:
Updated.